### PR TITLE
Add Retevis RT15 - fixes #6855

### PR DESCRIPTION
**Pull request by KC9HI**
# CHIRP PR Checklist

The following must be true before PRs can be merged:

* All tests must be passing.
* Commits should be squashed into logical units.
* Commits should be rebased (or simply rebase-able in the web UI) on current master. Do not put merge commits in a PR.
* Commits in a single PR should be related.
* Major new features or bug fixes should reference a [CHIRP issue](https://chirp.danplanet.com/projects/chirp/issues).
* New drivers should be accompanied by a test image in `tests/images` (except for thin aliases where the driver is sufficiently tested already).

Please also follow these guidelines:

* Keep cleanups in separate commits from functional changes.
* Please write a reasonable commit message, especially if making some change that isn't totally obvious (such as adding a new model, adding a feature, etc).
* Do not add new py2-compatibility code (No new uses of `six`, `future`, etc).
* All new drivers should set `NEEDS_COMPAT_SERIAL=False` and use `MemoryMapBytes`.
* New drivers and radio models will affect the Python3 test matrix. You should regenerate this file with `tox -emakesupported` and include it in your commit.

### CODE REVIEW

File: chirp/drivers/radtel_t18.py [hunk 1]
```diff
@@ -1020,7 +1034,7 @@ class RT22SRadio(T18Radio):
                     chirp_common.PowerLevel("Low",  watts=0.50)]
 
     _magic = b"9COGRAM"
-    _fingerprint = b"SMP558" + b"\x02"
+    _fingerprint = [b"SMP558" + b"\x02", ]
```
Comment (kk7ds) [inline chirp/drivers/radtel_t18.py:1037]: Convention would say that you should make a list of one be `[thing]` not `[thing, ]`. Some stricter pep8 checkers might fail you on this :)

Comment (KC9HI) [reply chirp/drivers/radtel_t18.py:1037]: OK. I just just seen quite a few lists done this way so I thought it was some convention or something. I will fix it.

Comment (kk7ds) [reply chirp/drivers/radtel_t18.py:1037]: Well, we might have more in the chirp code, but... in the grand scheme of cosmic python convention, it's unusual. No need to fix it necessarily, I'm just saying...

Comment (KC9HI) [reply chirp/drivers/radtel_t18.py:1037]: Oh.. I didn't notice that you had already merge it.

**Pull request merged**